### PR TITLE
Create TwilML parser

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+python:
+  - "2.6"
+  - "2.7"
+install:
+  - "pip install -r requirements.pip"
+  - "pip install -r requirements-dev.pip"
+  - "pip install coveralls"
+  - "python setup.py install"
+script:
+    - "coverage run --source=vumi_twilio_api `which py.test` vumi_twilio_api"
```
Comment (smn) [inline .travis.yml]: indent nitpick, I apologize for my OCD.

Comment (hodgestar) [inline .travis.yml]: Don't know why I can only see Simon dH's comment on the indenting in email and not GitHub.

**Comment by hodgestar** [comment]
:+1: on landing and iterating from here. A few general comments:
- We should perhaps use Twisted's trial.unittest instead of unittest since this is going to be Twisted based.
- It seems like there should be a cleaner way to assert on the exceptions (perhaps a combination of Twisted's `f = self.assertRaises(...)` or just a helper function).

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="vumi_twilio_api",
```
Comment (hodgestar) [inline setup.py]: I'm wondering if `vxtwilio` might be a better name?

Comment (hodgestar) [inline setup.py]: Also wondering whether using Twilio in the name might cause trademark issues? If so, maybe `vxtwinio`? :)

**Comment by rudigiesler** [comment]
@hodgestar I've addressed your comments. Ready for another review

**Comment by hodgestar** [comment]
:+1: !

**Pull request merged**